packages/ui/src/views/repo/pull-request/components/pull-request-item-description.tsx

```diff
@@ -9,6 +10,9 @@ interface PullRequestItemDescriptionProps {
   author: string
   sourceBranch: string
   timestamp: string
+  targetBranch: string
+  spaceId?: string
+  repoId?: string
```

Comment:
do we only need this for just navigation? Can we use a relative path and not have these props at all?

Reply:
done